Apeswap-Bots/New-Bonus-Multiplier/src/agent.ts

```diff
@@ -0,0 +1,20 @@
+import { Finding, HandleTransaction, TransactionEvent, ethers } from "forta-agent";
+import { contractType, APESWAP_MASTER_APE, UPDATE_MULTIPLIER_FUNCTION, createFinding } from "./utils";
+
+export function provideBotHandler(contractInfo: contractType): HandleTransaction {
+  return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+
+    const functionInvocations = txEvent.filterFunction(UPDATE_MULTIPLIER_FUNCTION, contractInfo.address);
+    functionInvocations.forEach((invocation) => {
+      const { args }: { args: ethers.utils.Result } = invocation;
+      findings.push(createFinding(`${args.multiplierNumber}`, contractInfo));
```

Comment:
```suggestion
      findings.push(createFinding(args.multiplierNumber.toString(), contractInfo));
```

this would also work